bin/cli.js

```diff
@@ -36,7 +41,6 @@ if (argv.version) {
 if (argv.help) {
   console.log('Usage: module-init [options]')
   clopts.print()
-  console.log('Run without arguments for interactive mode')
```

Comment:
ain't no non-interactive mode.
todo: `--force` option to run immediately with defaults.